makefile

```diff
@@ -1,6 +1,6 @@
 # You need rasm.exe in your PATH
 #
-RASM=rasm.exe
+RASM=/home/jorgegv/src/spectrum/rasm/rasm.exe
```

Comment:
There shouldn't be hard coded paths from your machine here right?

Reply:
right